### ISSUE TITLE
E-mail Alert not receiving from ossec server 3.6.0 on ubuntu 18.04.04

### ISSUE THREAD

**Issue by siddharth58** [ossec/ossec-hids#1859]
Hi, 
 i have installed ossec 3.6.0 on ubuntu 18.04.04
on that time of installation of ossec i have entered my email id and smtp details.

now trying to change active alert setting and frequency setting .
and also try to check smtp details but not successful..
need help..

**Comment by ddpbsd** [comment]
Can you check the mail server's logs to see why it isn't working?
Are there any maild logs in ossec.log?

**Comment by siddharth58** [comment]
hi 
thank you very much for your reply..
yes i jave check its showing error in smtp.
could you pls help me to resolve this , i'm new in this.

i have attached log file could pls guide me ..?
thank you :)
[ossec.txt](https://github.com/ossec/ossec-hids/files/4395180/ossec.txt)

**Comment by ddpbsd** [comment]
So OSSEC expects a 220 from the smtpd when it connects. It throws the banner error when it doesn't get that.

**Comment by siddharth58** [comment]
In this case should I change SMTP server config.or something else..

Thank you 😊

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ddpbsd** [comment]
I don't know how secureserver.net is setup. Look through their directions. I'm guessing you'll need auth though, so OSSEC will need some kind of relay to do that.

**Comment by siddharth58** [comment]
Hi,
thanks yes i have just mention smtp in email configuration.
could you pls help me to add relay on OSSEC.

**Comment by ddpbsd** [comment]
I don't think Ubuntu comes with an smtpd by default, so pick one and set it up as a relay.

**Comment by siddharth58** [comment]
yes, i have installed sendmail and try to get email alert on my domain but failed then i try postfix.
when i configure gmail account i get alerts but when i'm using my domain then its failed.
so i think its issue of our smtp server.

**Comment by atomicturtle** [comment]
Tagged as stale/closed if no further responses in 7 days
